src/pages/ServiceConfiguration.js

```diff
@@ -129,6 +129,17 @@ class ServiceConfiguration extends Component {
         label: translate('deploy.progress.ready-deployment'),
         playbooks: [constants.READY_DEPLOYMENT_PLAYBOOK + '.yml']
       }];
+      if(this.state.sesEnabled) {
```

Comment:
@eapodaca  looks like the only way to set sesEnabled true is to upload file. Once my file is there after I  refresh my page ..then sesEnabled is not true anymore. When I make changes to ses_configure file, it doesn' get into this block.

Reply:
After upload the file, I run the update, some of the steps are not updated.

Reply:
![image](https://user-images.githubusercontent.com/28358895/57107072-e7921f80-6ce3-11e9-9558-92d25777119b.png)

Reply:
This edge case causes problems even editing config files. If you edit a file and save it the update button is enabled. Refresh and its disabled, but the changes are not yet committed in the my_cloud repo.

Reply:
ok, refreshing page after load the ses config file might not be a normal use case. I am ok with it. However  if I don't refresh page, it shows the deploy and reconfigure steps in the progress...but those steps are not updated.

Reply:
@eapodaca  rerun the changes, now I understand why the steps are not updated. For Deploy SES step,  need to add the playbook into playbooksToRun.playbooks if his.state.sesEnabled. For reconfiguring cinder, glance, nova which I believe they are part of ardana reconfiguring, in the production I assume they are updated..however in development, they won't be updated unless you mock them.